### PR TITLE
fix: increase item_id max lenght from 128 to 255 for peer, staff and studenttraining workflow

### PR DESCRIPTION
**Pull request by vunguyen-dmt**
**TL;DR -** [ A short summary of what this PR does and why ]
The current max length for item_id fields of peerworkflow, staffworkflow and studenttrainingworkflow is 128 characters which are insufficient for course_id with a length greater than 70 characters. This PR alters the max length of item_id from 128 to 255.

**What changed?**

- Increase item_id max length of peerworkflow, staffworkflow and studenttrainingworkflow  from 128 to 255.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

- Check if the definitions of these fields are varchar(255). openedx.assessment_peerworkflow.item_id, openedx.assessment_staffworkflow.item_id and openedx.assessment_studenttrainingworkflow.item_id.
- Create a new course with an  ID > 70 characters.
- Create an Open Response component that has Student training and Peer and Staff steps.
- Submit responses and assess them.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @vunguyen-dmt! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by vunguyen-dmt** [comment]
Hi @pomegranited, I made a PR following up on [this post](https://discuss.openedx.org/t/report-open-response-edx-ora2-bug-cannot-submit-responses-for-courses-with-ids-greater-than-70-characters/12368/2), please take a look.

**Comment by pomegranited** [comment]
Thank you for submitting this @vunguyen-dmt ! Could you fix the commit lint error? (you'll need to use `git push --force` to push a new commit):

> Error: You have commit messages with errors
> ⧗   input: fix: increase item_id max lenght from 128 to 255 for peerworkflow, staffworkflow and studenttrainingworkflow
> ✖   header must not be longer than 100 characters, current length is 108 [header-max-length]
> ✖   found 1 problems, 0 warnings
> ⓘ   Get help: https://open-edx-proposals.readthedocs.io/en/latest/oep-0051-bp-conventional-commits.html

**Comment by pomegranited** [comment]
Hi @vunguyen-dmt , the error happened because your commit message's 1st line is too long. To fix it, you can do this:

```
# From your branch, run:
git commit --amend

# ^ command opens your editor.
# 1. Wrap the first line around to another line.
# 2. Save

# This pushes your updated commit message to this PR.
git push --force
```

**Comment by openedx-webhooks** [comment]
@vunguyen-dmt Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-ora2&entry.867055334=https://github.com/openedx/edx-ora2/pull/2183&entry.1484655318=https://github.com/vunguyen-dmt&entry.752974735=2024-03-01+11:29&entry.1917517419=2024-03-06+09:53&entry.2133058324=No) so we can improve your experience in the future.
